### PR TITLE
Buku forms

### PR DESCRIPTION
**Pull request by Klrfl**

### CODE REVIEW

**Comment by Adreon19** [comment]
conflict ygy
EllenJoeZenlessZoneZeroGIF (2)

**Review by Adreon19 — APPROVED** [review]
everything's fine and working as it should be. Manually importing the primevue made the Input finally showing up in my local. 🗣️🔥
Screenshot_2023-09-24_212352-removebg-preview

**Comment by Klrfl** [comment]
Mantab e konflik su tidak ada

**Comment by Klrfl** [comment]
tadi di formm, breakpoint nya kegedean kalau pake lg, pake sm aja ya

**Comment by Adreon19** [comment]
yes boss

**Comment by Adreon19** [comment]
Screenshot 2024-11-01 102837

**Review by Adreon19 — APPROVED** [review]
alldone

**Comment by Klrfl** [comment]
sekarang form edit buku

**Comment by Adreon19** [comment]
omw boss
Screenshot (25)

**Comment by Adreon19** [comment]
done boss, gua serahin fetch data nya ke lu
Screenshot (47)

**Comment by Adreon19** [comment]
udah belum

**Comment by Adreon19** [comment]
Screenshot 2023-07-08 223208

**Comment by Klrfl** [comment]
Belum bos

**Comment by Adreon19** [comment]
are we there yet

File: pages/admin/buku/[isbn].vue [hunk 1]
```diff
@@ -146,72 +147,136 @@ function toggleFormVisibility() {
...
+            <InputNumber
+              id="buku-jumlah"
+              v-model="buku.jumlah_exspl"
+              :invalid="0 > buku.jumlah_exspl || buku.jumlah_exspl > 1000"
+              name="buku-jumlah"
+              required
+              class="rounded-md border-gray-300 dark:border-gray-700 focus:ring-2 focus:ring-green-500 focus:outline-none"
+            />
+          </label>
+          <label for="buku-kategori" class="flex flex-col">
```
Comment (Klrfl) [inline pages/admin/buku/[isbn].vue]: sebenarnya label2nya ga usah dikasih `flex flex-col` sih tp gapapa

Comment (Adreon19) [reply pages/admin/buku/[isbn].vue]: sip

**Review by Klrfl — COMMENTED** [review]
Very small oversight

**Comment by Adreon19** [comment]
dont know what to say, man
Screenshot 2024-11-21 160000

**Pull request merged**